lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -189,8 +195,10 @@ def configure(conf)
         # Check for specialized GCE environments (Managed VM or Dataflow).
         # TODO: Add config options for these to allow for running outside GCE?
         attributes = fetch_gce_metadata('instance/attributes/').split
-        if attributes.include?('gae_backend_name') &&
-           attributes.include?('gae_backend_version')
+        if @force_compute_service
```

Comment:
Not really...  coming up with names is definitely the hardest part.  Definitely not 'kube' since it also affects MVM, dataflow, etc.  Maybe "detect_gce_subservice" defaulting to true?  Or, I'm fine leaving it as-is.